### ISSUE TITLE
[Liberté] ClassCastException: class VASSAL.build.module.properties.MutableProperty$Impl cannot be cast to class javax.swing.JButton (VASSAL.build.module.properties.MutableProperty$Impl is in unnamed module of loader 'app'; javax.swing.JButton is in module java.desktop of loader 'bootstrap')

### ISSUE THREAD

**Issue by uckelman** [vassalengine/vassal#11037]
| ABR | |
|-----|-----|
| Version | 3.6.3 |
| Reporter | markben61@gmail.com |
| Error Log | [68691 bytes](https://vassalengine.org/files/gh/7b9bce7208ac5c159743d88f76bb8dd7425e30ae) |

<pre>Removed the previously mentioned disabling property from the multiaction button. crash still occurs so may be the property isn't truely removed ?				

Liberté v1.0a08 3.6.3

java.lang.ClassCastException: class VASSAL.build.module.properties.MutableProperty$Impl cannot be cast to class javax.swing.JButton (VASSAL.build.module.properties.MutableProperty$Impl is in unnamed module of loader 'app'; javax.swing.JButton is in module java.desktop of loader 'bootstrap')
 at VASSAL.build.module.ToolbarMenu.propertyChange(ToolbarMenu.java:287)
 at java.desktop/java.beans.PropertyChangeSupport.fire(PropertyChangeSupport.java:343)
 at java.desktop/java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:335)
 at java.desktop/java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:268)
</pre>

**Comment by riverwanderer** [comment]
Note that I had restarted the module before applying the change. I then opened a new game and got the crash when I joined a playerside that would have set the GP in question to false.  However, after restarting the module and repeating the steps, the problem did not recur (but, for now I guess I can't use the button-lock function on a multi-action button).

It seems that the MA button is the problem, as I had an issue with MA and the lock-button property in a different area. (Duplicate of #11025. ?).
